try {
    console.log(naoExisto)
} catch(error) {
    console.log('Essa variável não existe.')
    console.log(error)
}